SwiftIOSExperiments/Network/AuthService.swift

```diff
@@ -6,20 +6,21 @@
 //  Copyright © 2017 James Santos. All rights reserved.
 //
 
-import Foundation
 import Auth0
 
-public struct Auth {
-	public var email: String
-	public var password: String
+struct Auth {
+	var email: String
+	var password: String
 
-	public init(email: String, password: String) {
+	init(email: String, password: String) {
 		self.email = email
 		self.password = password
 	}
 }
 
 class AuthService {
+	static var token: String!
+
```

Comment:
This is temporary, I was planning to use SimpleKeychain to manage session/auth tokens.